purchase_order_reorder_lines/tests/test_invoice_on_lines.py

```diff
@@ -27,6 +27,8 @@
 
 class testPurchasePartialInvoicing(common.TransactionCase):
 
+    post_install = True
```

Comment:
Well, I was saying the contrary... Can you start putting the stack trace of the error, for knowing what is happening?

Reply:
Ok, sorry.

Stack trace when running test is:
`ValueError: Invalid field 'fully_invoiced' in leaf "<osv.ExtendedLeaf: ('fully_invoiced', '=', False) on purchase_order_line (ctx: )>"`

Reply:
I'm afraid that's not enough. Where is that risen? As we are doing this as inherited and temporary while we migrate to 13.0 version, I'm thinking of just disabling the execution of this test in our integration CI.